### DISCUSSION TITLE
Juniper Plugin

### DISCUSSION THREAD

**Discussion by dan-dev14** [dmulyalin/fakenos#3] [Q&A]
Hi,

I'm creating a Juniper plugin for FakeNOS as I couldn't find one (correct me if I'm wrong!). 

What would I need to set initial prompt to in order to match Juniper and pull the user attribute in there too? e.g. "jcluser@R1>"

Currently I've just got the default:
`
name = "juniper_junos"
initial_prompt = "{base_prompt}"
`

**Comment by dmulyalin** [comment]
probably this:
```
initial_prompt  = "jcluser@{base_prompt}>"
```

But username would not be replaced automatically, FakeNOS does not support that but probably can be a good feature request to add support for something like:
```
initial_prompt  = "{user_name}@{base_prompt}>"
```

For further details on creating NOS plugin please refer to [docs](https://dmulyalin.github.io/fakenos/creating_nos_plugin/)

**Reply by dan-dev14** [reply]
Yeah apologies, adding it dynamically is what I meant, setting it manually is fine for now. Definitely would be a nice feature to have.
